python/functional-programming/functional-programming/why-functional-programming-ii.md

```diff
@@ -0,0 +1,49 @@
+---
+author: stefkn
+
+levels:
+  - beginner
+  - basic
+  - medium
+
+aspects:
+  - introduction
+  - obscura
+
+type: normal
+
+category: must-know
+
+links:
+  - '[Advantages Of Functional Programming](http://wiki.c2.com/?AdvantagesOfFunctionalProgramming){website}'
+  - '[Functional Programming HOWTO](https://docs.python.org/2/howto/functional.html){website}'
+
+
+---
+
+# Why Functional Programming? (Part 2)

```

Comment:
since the insight has only one heading, we could change the headline to that
```suggestion
# Eschewing Shared State
```

followed by removing the heading